.ci-config/rippled.cfg

```diff
@@ -167,3 +163,7 @@ F1ED6B4A411D8B872E65B9DCB4C8B100375B0DD3D62D07192E011D6D7F339013 fixTrustLinesTo
 # 1.12.0-b1 Amendments
 56B241D7A43D40354D02A9DC4C8DF5C7A1F930D92A9035C4E12291B3CA3E1C2B featureClawback
 27CD95EE8E1E5A537FF2F89B6CEB7C622E78E9374EBD7DCBEDFAE21CD6F16E0A fixReducedOffersV1
+# Added August 9th, 2023
+86E83A7D2ECE3AD5FA87AB2195AE015C950469ABF0B72EAACED318F74886AE90 CryptoConditionsSuite
+AE35ABDEFBDE520372B31C957020B34A7A4A9DC3115A69803A44016477C84D6E fixNFTokenRemint
+B2A4DB846F0891BF2C76AB2F2ACC8F5B4EC64437135C6E56F3F859DE5FFD5856 ExpandedSignerList
```

Comment:
Should probably be added above since it was much older. Added in 1.9.1.

Reply:
Might need to add an integration test for `WalletLocator` and the size of SignerLists

Reply:
Would it be alright to leave that for a separate ticket? (I'm not exactly sure what we'd be testing for that - maybe you can flesh out an Issue to add it w/ an idea for what we'd want to test on the client library side?)